app/models/school.rb

```diff
@@ -4,4 +4,16 @@ class School < ApplicationRecord
   belongs_to :network, optional: true
   has_one :partnership
   has_one :lead_provider, through: :partnership
+  has_and_belongs_to_many :induction_coordinator_profiles
+
+  def full_address
+    address = <<~ADDRESS
```

Comment:
I think we should add a unit test for this. We don't need to assign it to address if we are just building and returning the string